api.go

```diff
@@ -801,6 +801,8 @@ func importExistenceColumns(index *Index, columnIDs []uint64) error {
 }
 
 // MaxShards returns the maximum shard number for each index in a map.
+// TODO (2.0): This method has been deprecated. Instead, use
+// AvailableShardsByIndex.
```

Comment:
is it possible now to remove the `/internal/shards/max` endpoint?

Reply:
That's a good question. I'll check to see if any of the clients are still using that.